src/main/java/magicbees/main/utils/compat/BloodHelper.java

```diff
@@ -0,0 +1,145 @@
+package magicbees.main.utils.compat;
+
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemStack;
+
+import WayofTime.alchemicalWizardry.ModItems;
+import WayofTime.alchemicalWizardry.api.altarRecipeRegistry.AltarRecipeRegistry;
+import cpw.mods.fml.common.Loader;
+import cpw.mods.fml.common.registry.GameRegistry;
+import cpw.mods.fml.relauncher.Side;
+import cpw.mods.fml.relauncher.SideOnly;
+import magicbees.main.CommonProxy;
+import magicbees.main.Config;
+import magicbees.main.utils.compat.bloodmagic.ItemBloodBaseFrame;
+import magicbees.main.utils.compat.bloodmagic.ItemBloodFrame;
+import magicbees.main.utils.compat.bloodmagic.ItemFrenziedFrame;
+import thaumcraft.api.ThaumcraftApi;
+import thaumcraft.api.aspects.Aspect;
+import thaumcraft.api.aspects.AspectList;
+import thaumcraft.api.crafting.IArcaneRecipe;
+import thaumcraft.api.research.ResearchCategories;
+import thaumcraft.api.research.ResearchItem;
+import thaumcraft.api.research.ResearchPage;
+import thaumcraft.common.config.ConfigItems;
+
+public class BloodHelper implements IModHelper {
+
+    public static Item hiveFrameBlood;
+    public static Item hiveFrameFrenzy;
+    public static Item hiveFrameBloodBase;
+
+    @SideOnly(Side.CLIENT)
```

Comment:
What's this for?

Reply:
Leftover from something deleted, it's been removed